docs/01_nodeos/03_plugins/producer_plugin/10_block-producing-explained.md

```diff
@@ -4,87 +4,61 @@ content_title: Block Production Explained
 
 For simplicity of the explanation let's consider the following notations:
 
-m = max_block_cpu_usage
+* `r` = `producer_repetitions = 12` (hard-coded value)
+* `m` = `max_block_cpu_usage` (on-chain consensus value)
+* `n` = `max_block_net_usage` (on-chain consensus value)
+* `t` = `block-time`
+* `e` = `produce-block-offset-ms` (nodeos configuration)
+* `w` = `block-time-interval = 500ms` (hard-coded value)
+* `a` = `produce-block-early-amount = w - (w - (e / r)) = e / r ms` (how much to release each block of round early by)
+* `l` = `produce-block-time = t - a`
+* `p` = `produce block time window = w - a` (amount of wall clock time to produce a block)
+* `c` = `billed_cpu_in_block = minimum(m, w - a)`
+* `n` = `network tcp/ip latency`
+* `h` = `block header validation time ms`
+
+Peer validation for similar hardware/version/config will be <= `m`
+
+**Let's consider for exemplification the following two BPs and their network topology as depicted in the below diagram**
```

Comment:
Why `exemplification` is used here instead of just `for example` is beyond me. Seems like we are trying to sound smart.

Reply:
I think I'd write:
`Let's consider the exemple of the following two BPs and their network topology as depicted in the below diagram`